LOAN/LoanContracts/LoanCore.rdf

```diff
@@ -1146,35 +1065,36 @@
 As needed, use the worksFor property to connect appraisers to the appraisal management company.</fibo-fnd-utl-av:usageNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-ln;ReverseMortgageLoanContract">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanContract"/>
+	<owl:Class rdf:about="&fibo-loan-ln-ln;ReverseMortgage">
+		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;Mortgage"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-loan-ln-ln;hasCreditLimit"/>
 				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>reverse mortgage loan contract</rdfs:label>
+		<rdfs:label>reverse mortgage</rdfs:label>
 		<skos:definition>a mortgage contract that pays out money to the borrower against a set principal limit that is based on the value of existing equity in the underlying collateral.</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote>A reverse mortgage and an open end loan both have a credit limit.</fibo-fnd-utl-av:explanatoryNote>
 		<fibo-fnd-utl-av:explanatoryNote>The interest is added to the principal balance.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
The note does not make much sense from a FIBO point of view.

Reply:
Ok - we need to do several more passes on this in any case